### ISSUE TITLE
tag_image skip_when_tags_exist does not work

### ISSUE THREAD

**Issue by swiknaba** [CircleCI-Public/aws-ecr-orb#297]
### Orb version

9.0.0

### What happened

```yml
  - aws-ecr-latest-broken/tag_image:
      repo: << pipeline.parameters.project-name >>
      source_tag: 'commit-${CIRCLE_SHA1}'
      target_tag: 'latest-${CIRCLE_BRANCH//\//-}'
      skip_when_tags_exist: false
```

setting `skip_when_tags_exist` ends in:

```
   "imageTag": "latest-production"
+ echo 'Tag "latest-production" already exists and will be skipped.'
Tag "la****-production" already exists and will be skipped.
+ set +x
```

### Expected behavior

not skipping when a tag already exists should push the tag anyways. 

Why: I want to always add a 2nd tag to the image that is currently deployed.

**Comment by iusztin** [comment]
Looks like the `true` and `false` values were mixed up?

> Boolean values may be returned as a ‘1’ for True and ‘0’ for False.
https://circleci.com/docs/reusing-config/#boolean

```bash
...
    # if skip_when_tags_exist is true
    if [ "${ORB_BOOL_SKIP_WHEN_TAGS_EXIST}" -eq "0" ]; then
 ...
```
See https://github.com/CircleCI-Public/aws-ecr-orb/blob/v9.0.0/src/scripts/tag_image.sh#L14

**Comment by brivu** [comment]
This issue was resolved in #305. Thank you for letting us know!
